lib/templates.php

```diff
@@ -96,19 +114,19 @@ function gutenberg_register_template_post_type() {
  * @param array $post    Template Post.
  * @param bool  $update  Is update.
  */
-function gutenberg_set_template_post_theme( $post_id, $post, $update ) {
-	if ( 'wp_template' !== $post->post_type || $update || 'trash' === $post->post_status ) {
+function gutenberg_set_template_and_template_part_post_theme( $post_id, $post, $update ) {
+	if ( $update || 'trash' === $post->post_status ) {
 		return;
 	}
```

Comment:
Can we remove this early return? This is basically what I was asking for originally, not so much a migration. I don't believe there is any case in which a template should be without a theme, not with the current implementation anyway.

Reply:
Plus, it would only ever happen through a corruption of the database, since the theme is automatically added upon the post first being created. Removing this early return would just protect against that.

Reply:
Sounds right to me! Committing